internal/postprocessor/main.go

```diff
@@ -96,3 +147,149 @@ func copyFiles(srcPath, dstPath string) error {
 
 	return nil
 }
+
+func ParseAPIShortnames(googleapisDir string, confs []*generator.MicrogenConfig, manualEntries []generator.ManifestEntry) (map[string]string, error) {
```

Comment:
Please add some docs to this function. Also, should this be private?

Reply:
Maybe also metion in a dev comment this was copied from somewhere else?

Reply:
I removed it. I had copied because of dependency issues that wasn't letting me call it before, but it works to call it from gapicgen/generator/gapics.go now.